packages/wdio-cucumber-framework/src/utils.js

```diff
@@ -160,3 +168,96 @@ export function getStepFromFeature(feature, pickle, stepIndex, sourceLocation) {
 
     return targetStep
 }
+
+/**
+ * @param {object[]} result cucumber global result object
+ */
+export const getDataFromResult = ([{ uri }, feature, ...scenarios]) => ({ uri, feature, scenarios })
+
+/**
+ * wrap every user defined hook with function named `userHookFn`
+ * to identify later on is function a step, user hook or wdio hook.
+ * @param {object} options `Cucumber.supportCodeLibraryBuilder.options`
+ */
+export function setUserHookNames (options) {
+    const hooks = [
+        'beforeTestRunHookDefinitions',
+        'beforeTestCaseHookDefinitions',
+        'afterTestCaseHookDefinitions',
+        'afterTestRunHookDefinitions',
+    ]
+    hooks.forEach(hookName => {
```

Comment:
```suggestion
    hooks.forEach(hookName => options[hookName].forEach(testRunHookDefinition => {
```

Reply:
@christian-bromann isn't it conflicting with what @erwinheitzman suggested? 
See https://github.com/webdriverio/webdriverio/pull/4288#discussion_r312802136

I might be missing something

Reply:
yeah, it is a code style thing where we don't have any eslint rules enforcing anything. I personally like if we can save indentation if possible and where it makes sense. I don't have a hard opinion on this.